src/platform/Zephyr/BUILD.gn

```diff
@@ -71,6 +74,13 @@ static_library("Zephyr") {
     }
   }
 
+  if (chip_mdns == "minimal") {
```

Comment:
This doesn't make sense. "minimal" is the implementation that is platform-agnostic and doesn't use `ChipDnssdXXX` APIs at all. If you use "minima", you don't need any Zephyr-specific bits except for multicast support. `ChipDnssdXXX` APIs are only relevant for chip_mdns="platform".

Reply:
Is your device Thread-based? In some issue you were asking about Ethernet driver.

Reply:
My device is Ethernet-based. I only use Zephyr SDK, no ST Microelectronics things.